src/main/java/com/google/opensesame/auth/AuthServlet.java

```diff
@@ -59,14 +66,19 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       userData.addProperty("id", currentUser.getUserId());
 
       responseObject.add("user", userData);
+
+      // TODO : Add ability to check that a PersonObject exists in datastore with the user's ID.
+      // For now, will assume to be false. This can be implemented when datastore support is added
+      // for the PersonObject
+      // Checking the datastore for an existing PersonObject is mildly inefficient. This is a side
+      // effect of having to use the Users API on top of GitHub OAuth. Because there are two steps
+      // in signing up, this check ensures that the user has completed both authentication steps.
+      // If in the future OAuth can be used for backend authentication (through something like the
+      // Firebase Auth Admin SDK), this check can be removed (along with all Users API code).
```

Comment:
```suggestion
      // If in the future OAuth can be used for backend authentication (through something like the
      // Firebase Auth Admin SDK), we can remove the logic to check the datastore for an existing
      // PersonObject with the user's ID (along with all Users API code).
```
I don't understand the first 3 sentences in the second paragraph. The term `mildly inefficient` is vague. When I read it I think it could mean either that datastore is slow or checking for an existing PersonObject shouldn't be necessary. 

Also `This is a side effect of having to use the Users API on top of GitHub OAuth` confuses me because I don't know what `this` is. In general don't use the word `this` in technical documentation; explicitly state what you are talking about.

I think it would be sufficient to just leave the last 2 sentences but replacing the word `this` with more specific language.

Reply:
These are good points. I'll be more careful about vagueness and ambiguous terms like `this` moving forward. I like your suggested change as well, thanks!